elements/lisk-p2p/src/peer/base.ts

```diff
@@ -93,7 +93,7 @@ export const REMOTE_EVENT_MESSAGE = 'remote-message';
 
 export const REMOTE_RPC_UPDATE_PEER_INFO = 'updateMyself';
 export const REMOTE_RPC_GET_NODE_INFO = 'status';
-export const REMOTE_RPC_GET_ALL_PEERS_LIST = 'list';
+export const REMOTE_RPC_GET_MINIMAL_PEERS_LIST = 'minimalList';
```

Comment:
why we are giving a new RPC name? we have `getPeers` RPC call that has the same requirement.

Reply:
We're keeping them separated. I'll rename back the `REMOTE_RPC_GET_ALL_PEERS_LIST` to make it clear.